packages/manager/src/features/CloudPulse/Utils/FilterBuilder.ts

```diff
@@ -313,8 +313,10 @@ export const buildXFilter = (
       }
     });
   }
-
-  return { '+and': filters, '+or': orCondition };
+  if (orCondition.length > 0) {
```

Comment:
```suggestion
  if (orCondition.length) {
```

why can't we use it like this